src/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, agentName, loadName, baseAgentName=False, properties=None):
     def initialize(self):
         """Sets default parameters and creates CE instance"""
 
+        self.am_setOption("MonitoringEnabled", False)
```

Comment:
Does it make sense to have a different default with respect to the `Agent` ?

Reply:
JobAgent(s) should not send monitoring data (maybe I have not fully understood your question...?)

Reply:
Ah okay, I did not know always wanted to disable it. Scratch that then :-)